_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -5569,11 +5567,7 @@
 /turf/open/indestructible/hotelwood,
 /area/centcom/holding/cafepark)
 "aXd" = (
-/obj/machinery/vending/clothing{
-	default_price = 0;
-	extended_inventory = 1;
-	vend_reply = "Thank! Enjoy your stay at the Cafe."
-	},
+/obj/machinery/vending/access/command,
```

Comment:
Did you check that they actually have the access requited for this to work at all?

Reply:
i mean, they can access the other vendors just fine, I think their janitor pass is some sort of AA